Source/Data/EventAnalysis/KTProcessedTrackData.hh

```diff
@@ -33,6 +33,9 @@ namespace Katydid
 
             MEMBERVARIABLE(bool, IsCut);
 
+            MEMBERVARIABLE(double, MVAClassifier);
```

Comment:
The older MVA classifier modifies the track object; the new DLIB classifier creates a new data object with more detailed classification info

Reply:
Hmm, I see.  This does introduce possible confusion, and it might be worth moving it to a new data type.  But let's leave it for now and focus on items with higher priorities.

Reply:
I agree, it might be worth making them operate the same way in the future.